packages/app/src/themes/componentOverrides.ts

```diff
@@ -10,8 +11,50 @@ const redhatFont = `@font-face {
     url(/fonts/RedHatText-Regular.ttf) format('truetype');
 }`;
 
-export const components: UnifiedThemeOptions['components'] = {
-  MuiCssBaseline: {
-    styleOverrides: redhatFont,
-  },
+export const components = (
+  themeColors: ThemeColors,
+): UnifiedThemeOptions['components'] => {
+  return {
+    BackstageHeaderTabs: {
+      styleOverrides: {
+        tabsWrapper: {
+          paddingLeft: '0px',
+        },
+      },
+    },
+    MuiTabs: {
+      defaultProps: {
+        TabIndicatorProps: {
+          style: {
+            height: '3px',
+            background: themeColors.navigationIndicatorColor || '#0066CC',
+          },
+        },
+      },
+      styleOverrides: {
+        root: {
+          borderBottom: '1px solid #d2d2d2',
+        },
+      },
+    },
+    MuiTab: {
+      defaultProps: {
+        disableRipple: true,
+      },
+      styleOverrides: {
+        root: {
+          minWidth: 'auto',
```

Comment:
Unluckily this minWidth style is ignored because there is another style based on another break.

I would recommend this for now:

```suggestion
          minWidth: 'initial !important',
```

Or do you know how to define breakpoints here? I don't.